__main__.py

```diff
@@ -1,3 +1,5 @@
 #from wsgi.py import application as app
 # This is to meet GLoud standard of having "main.py" file
-from app.__init__ import *
\ No newline at end of file
+from app import app
+
+app.run(debug=True)
```

Comment:
`__main__` will now explicitly start the app itself. This allows for the testing init stuff to import `app` without it starting the app.